<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background_wall">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>-5</y>
     <width>701</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(130, 123, 117)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="xp_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>XP :</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="inn_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; self.deathbar_hero.setPixmap(QPixmap(hero_empty))
                    hero_full = os.path.join(os.path.dirname(__file__), &quot;..&quot;,
                    &quot;ressources&quot;, &quot;full_health.png&quot;)
                    self.lifebar_hero.setPixmap(QPixmap(hero_full))
                    self.deathbar_hero.setScaledContents(True)
                    self.lifebar_hero.setScaledContents(True)</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Inn - regain all your health - Costs 5 gold</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="sleep_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Sleep at the inn</string>
   </property>
  </widget>
  <widget class="QLabel" name="combat_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; self.deathbar_hero.setPixmap(QPixmap(hero_empty))
                    hero_full = os.path.join(os.path.dirname(__file__), &quot;..&quot;,
                    &quot;ressources&quot;, &quot;full_health.png&quot;)
                    self.lifebar_hero.setPixmap(QPixmap(hero_full))
                    self.deathbar_hero.setScaledContents(True)
                    self.lifebar_hero.setScaledContents(True)</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Combat</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_level1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0xp needed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_level2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100xp needed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_level3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>200xp needed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_level4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>300xp needed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_level5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>400xp needed</string>
   </property>
  </widget>
  <widget class="QLabel" name="gold_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; self.deathbar_hero.setPixmap(QPixmap(hero_empty))
                    hero_full = os.path.join(os.path.dirname(__file__), &quot;..&quot;,
                    &quot;ressources&quot;, &quot;full_health.png&quot;)
                    self.lifebar_hero.setPixmap(QPixmap(hero_full))
                    self.deathbar_hero.setScaledContents(True)
                    self.lifebar_hero.setScaledContents(True)</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>GOLD :</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="att_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; self.deathbar_hero.setPixmap(QPixmap(hero_empty))
                    hero_full = os.path.join(os.path.dirname(__file__), &quot;..&quot;,
                    &quot;ressources&quot;, &quot;full_health.png&quot;)
                    self.lifebar_hero.setPixmap(QPixmap(hero_full))
                    self.deathbar_hero.setScaledContents(True)
                    self.lifebar_hero.setScaledContents(True)</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>ATT :</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name_entry">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="class_entry">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="xp_entry">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="att_entry">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="gold_entry">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="deathbar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lifebar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: green;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="level_entry">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="hp_entry">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; </string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>491</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(184, 107, 53);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>491</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(111, 38, 38);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>641</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(103, 103, 103)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="inn_error">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(165, 225, 175);
                    self.deathbar_hero.setPixmap(QPixmap(hero_empty))
                    hero_full = os.path.join(os.path.dirname(__file__), &quot;..&quot;,
                    &quot;ressources&quot;, &quot;full_health.png&quot;)
                    self.lifebar_hero.setPixmap(QPixmap(hero_full))
                    self.deathbar_hero.setScaledContents(True)
                    self.lifebar_hero.setScaledContents(True)</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>background_wall</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>xp_label</zorder>
  <zorder>inn_label</zorder>
  <zorder>sleep_btn</zorder>
  <zorder>combat_label</zorder>
  <zorder>btn_level1</zorder>
  <zorder>btn_level2</zorder>
  <zorder>btn_level3</zorder>
  <zorder>btn_level4</zorder>
  <zorder>btn_level5</zorder>
  <zorder>gold_label</zorder>
  <zorder>att_label</zorder>
  <zorder>name_entry</zorder>
  <zorder>class_entry</zorder>
  <zorder>xp_entry</zorder>
  <zorder>att_entry</zorder>
  <zorder>gold_entry</zorder>
  <zorder>deathbar</zorder>
  <zorder>lifebar</zorder>
  <zorder>level_entry</zorder>
  <zorder>hp_entry</zorder>
  <zorder>inn_error</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
